force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,9 +32,15 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        
+        if(a != null && b != null){
+            Integer total = a + b;
+            return total;
+        }
```

Comment:
The closing bracket for the if and 'else {' should be on one line like this:
`} else {`